base/base/errnoToString.cpp

```diff
@@ -3,10 +3,11 @@
 #include <fmt/format.h>
 
 
-std::string errnoToString(int code, int the_errno)
```

Comment:
The usage of the "code" argument was strange and misinterpreted by the callers. I've removed it.